platform/cavium/docker-syncd-cavm/Dockerfile.template

```diff
@@ -0,0 +1,18 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY deps /deps
+
+RUN apt-get -y install libpcap-dev libxml2-dev python-dev swig libsensors4-dev redis-tools
```

Comment:
I don't see that other vendors install redis-tools into docker-syncd
Anyway, if you need it, redis is built from sources, so you should install it like [here](https://github.com/Azure/sonic-buildimage/blob/master/rules/docker-database.mk#L5)